internal/backup/volumesnapshotcontent_action.go

```diff
@@ -59,60 +59,69 @@ func (p *VolumeSnapshotContentBackupItemAction) Execute(item runtime.Unstructure
 		return nil, nil, errors.WithStack(err)
 	}
 
-	_, snapshotClient, err := util.GetClients()
-	if err != nil {
-		return nil, nil, errors.WithStack(err)
-	}
+	additionalItems := []velero.ResourceIdentifier{}
 
-	// Wait for VSC to be in ready state
-	VSCReady, err := util.WaitForVolumeSnapshotContentToBeReady(snapCont, snapshotClient.SnapshotV1(), p.Log)
+	if util.DataMoverCase() {
```

Comment:
suggestion: Add a tiny comment explaining why we are doing this.. otherwise lgtm

Reply:
Done, Thanks !